### DISCUSSION TITLE
USE_WEBENGINE by default

### DISCUSSION THREAD

**Discussion by trufanov-nok** [u-235/kchmviewer#11] [Ideas]
Would be great if USE_WEBENGINE will be enabled by default. My up-to-date Kubuntu 21.10 (Qt 5.15.2) doesn't compile the project without it. I suspect most of the current systems have Qt 5.9+ onboard.

**Comment by u-235** [comment]
This flag enables building the application with WebEngine instead of WebKit. I know that WebKit is considered outdated technology, but `libqt5webkit5-dev` is in actual [Debian](https://packages.debian.org/bookworm/libqt5webkit5-dev) and [Ubuntu](https://packages.ubuntu.com/impish/libqt5webkit5-dev) distributives, so for now this flag is off by default.
